spec/latest/json-ld-framing/index.html

```diff
@@ -291,7 +291,7 @@ <h3>Framing</h3>
       using an example <a>frame</a> document which is used to both match the
       <a data-cite="JSON-LD11CG-API#dfn-flattened">flattened</a>
       data and show an example of how the resulting data should be shaped.
-      Matching is performed by using <a>properties</a> present in in the <a>frame</a>
+      Matching is performed by using <a>properties</a> present in the <a>frame</a>
       to find objects in the data that share common values. Matching can be done
       either using all properties present in the frame, or any property in the frame.
```

Comment:
might as well fix two at once...
```suggestion
      using either all properties present in the frame, or any property in the frame.
```